### DISCUSSION TITLE
Why Tree sequences generated from Relate (convert .mut/.anc) has no population labels

### DISCUSSION THREAD

**Discussion by itsmestewart** [tskit-dev/tskit#2864] [Q&A] [answered]
I have generated a tree sequence using .mut/.anc generated by Relate.

When loaded using tskit.load, the resulting tree sequence has no population labels.
What could be the reason for this?

Note: The original input files have population labels.
Many thank for your assistance.

Please see the attached picture
<img width="472" alt="Screenshot 2023-10-12 at 11 15 23" src="https://github.com/tskit-dev/tskit/assets/73900141/073cb451-ae83-404b-8809-1aa38a1b4224">

**Comment by jeromekelleher** [comment, marked as answer]
Hi @itsmestewart, welcome :wave:

I think this is a question for the Relate developers rather than tskit, as it (Relate) performs the format conversion. I don't know the details of how Relate does this.

**Reply by itsmestewart** [reply]
hello @jeromekelleher ,thanks 

Yes, this is an isssue with Relate package, I did not get a active platform like this to adress this.  I appreciate your response.

**Comment by jeromekelleher** [comment]
Answer from @nspope on same question elsewhere:

> Yup -- looks like Relate's converter tool just doesn't construct the individual / population tables. There's no reason why it shouldn't do this (it'd only require parsing an additional input file; there's even a placeholder in the code for it). Would definitely be worth adding this feature (but that's on the Relate side of things, not tskit).

**Reply by itsmestewart** [reply]
Thanks
